### PR TITLE
Improve the Audit Trails views

### LINKED ISSUE #17105: Track the creation and modification dates, along with the user IDs of the creator and modifier, for each User instance.
I encountered an issue in production where I needed to determine when a user was created and when their information was last modified. Unfortunately, I discovered that this information was not being captured.

I believe it’s crucial to record the creation date and the user who created it. Additionally, storing the last modified date and the user who made the changes would provide valuable insights for tracking and auditing purposes.

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Fix #17105

### CODE REVIEW

**Comment by Piedone** [comment]
These details are already tracked by Audit Trail.

**Comment by MikeAlhayek** [comment]
So we have a way to view the user object as we do in content? I am not sure. 

It's useful to that that info in the user object as we do in contentItems

**Comment by Piedone** [comment]
This can be useful if the properties are included in `UserIndex` too, so one can filter/order on them. Without that, I don't see the benefit of duplicating these without the rich contextual info that Audit Trail provides.

**Comment by MikeAlhayek** [comment]
How do you access the create and modified user using the audit trails? I thought that the Users Audit trail would have revision control, but did not see a way to do it.

**Comment by Piedone** [comment]
The last modified, created dates for content items are there for historical reasons (these predate Audit Trail) and because it's useful to query on them. I think it's the same for Users. Why would we just store the same data that's already stored in Audit Trail, and better? And then not make querying available, for which they'd be actually required (for e.g. "list the most recently registered users first" or "list users who registered in November").

Having a separate section for the general Event Info and the specific one below for Audit Trail events is nice, but I don't get why duplicate things for users:
- Created at UTC should be the same as the event timestamp.
- Created by user ID is in the event info.
- The Modified* ones are the same, and we shouldn't display fields with blank values for different events (since there's also a User Updated event, recorded with all the basic details too).

**Comment by MikeAlhayek** [comment]
@Piedone okay. I dropped the CreatedUtc, CreatedBy, ModifiedUtc and ModifiedBy. Instead, I updated the trail audit views so they are grouped in such a way someone will understand what the values mean. I think with this format, It's easier to determine who did what.

image

image

**Comment by Piedone** [comment]
But why not also keep them, just add them to `UserIndex`? I do think with that, it's useful (but not without).

**Comment by MikeAlhayek** [comment]
Why would we inflate a index with data that isn't used?

Also, @sebastienros kinda mention in the meeting today that he is also not a fan of these properties as you originally suggested

File: src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventDetail.DetailAdmin.cshtml [hunk 1]
```diff
@@ -1,29 +1,31 @@
 @model AuditTrailEventViewModel
 
-<table class="table table-sm table-hover mb-0">
+<h3>@T["Event Info"]</h3>
```
Comment (Piedone) [inline src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventDetail.DetailAdmin.cshtml:3]: What's above this on the page is `h1`, so this should be an `h2`. Same for Content Item Info and User Info.

**Review by Piedone — CHANGES_REQUESTED** [review]
Because querying on users by these is a use case I can imagine being useful. But not insisting on it if we throw out the properties anyway.

File: src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventDetail.DetailAdmin.cshtml (hunk 1 above)
Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventDetail.DetailAdmin.cshtml:3]: it's makes the font way too large.

Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventDetail.DetailAdmin.cshtml]: I changed to to `<h2 class="h3"></h2>`

**Review by Piedone — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventDetail.DetailAdmin.cshtml (hunk 1 above)
Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventDetail.DetailAdmin.cshtml]: That's what a Hungarian saying calls "iron hoop made of wood" :D.

**Pull request merged**